CodeEditModules/Modules/GitClient/Tests/GitClientTests.swift

```diff
@@ -10,6 +10,8 @@ import ShellClient
 import XCTest
 
 final class GitClientTests: XCTestCase {
+    // FIXME: Update GitClient.getCommitHistory implementation on date or this test case
```

Comment:
Sorry could you move this `FIXME` above the `getCommitHistory` API instead of having it in the tests? The comment below makes perfect sense to have here but this one might be better on the actual API itself so it gets more visibility. Thank you!